### ISSUE TITLE
Config parser plotting scripts cannot handle IFO tags

### ISSUE THREAD

**Issue by GarethCabournDavies** [gwastro/pycbc#4381]
Say for example we want to set the stat-threshold for plot_binnedhist to be different for h1, l1 and v1. It would make sense to use:

```
[plot_binnedhist-h1]
stat-threshold = 5.0

[plot_binnedhist-l1]
stat-threshold = 5.5

[plot_binnedhist-v1]
stat-threshold = 6.0
```

However this causes an error in the generation:
```
ValueError: The following options appear in both section plot_binnedhist-h1 and plot_binnedhist-h1: stat-threshold
```

Which makes no sense to me - the section is only there once?

I've tried adding the ifo from the trigger file into the tags in [this line](https://github.com/gwastro/pycbc/blob/v2.2.0/pycbc/workflow/plotting.py#L454) in the plotting workflow module, and [this line](https://github.com/gwastro/pycbc/blob/v2.2.0/bin/workflows/pycbc_make_coinc_search_workflow#L401) in the coinc workflow generator, but still getting the same error

I can't work out if this is a problem from the config side or the way it is called, I can do this to (e.g.) `fit_by_template` fine

**Comment by spxiwh** [comment]
To be clear on this. The ifo should *not* be in the list of tags, as it is sent already in the `ifos`, and so would be treated as a tag from there. Is this what is happening (if so, some sanity checking could be added, but want to diagnose first).

**Comment by GarethCabournDavies** [comment]
The code doesn't get to there to fail in the case that the option is added.

When it isn't, `tags` is `['full_data']` and `secs` is `['mtotal']`, for all IFOs.

**Comment by GarethCabournDavies** [comment]
Just to add, it is not just this executable, it is:
plot_hist: `['h1', 'l1', 'v1'], ['full_data']`
plot_singles: `['mtotal_eta_newsnr', 'h1', 'l1', 'v1'], ['full_data']`

the sngl_minifollowups also cannot take ifo tags in, tried:
```
[workflow-sngl_minifollowups]
[workflow-sngl_minifollowups-h1]
num-sngl-events=1
[workflow-sngl_minifollowups-l1]
num-sngl-events=2
[workflow-sngl_minifollowups-v1]
num-sngl-events=3

[workflow-sngl_minifollowups-noncoinc]
section-header = loudest_noncoinc_time

[workflow-sngl_minifollowups-all]
section-header = all
```

and got:
`configparser.NoOptionError: No option 'section-header' in section: 'workflow-sngl_minifollowups-h1'`, with the `workflow.cp.get_subsections('workflow-sngl_minifollowups')` loop [here](https://github.com/gwastro/pycbc/blob/v2.2.0/bin/workflows/pycbc_make_coinc_search_workflow#L419) using `['h1', 'l1', 'v1', 'noncoinc', 'all']`

So I think there may be an underlying non-treatment of IFO tags

**Comment by spxiwh** [comment]
Okay, can you point me in the direction of a few failing examples that I can reproduce and look into? This might be more than one issue.

**Comment by GarethCabournDavies** [comment]
The options I pasted above are the examples which I could see which failed, I don't know of any others

**Comment by GarethCabournDavies** [comment]
This has been solved by #4402
